docs/index.md

```diff
@@ -23,13 +23,12 @@ that would not be covered by Grafonnet already.
 
 ## Scope
 
-Grafonnet aims at supporting any basic feature of dashboards (annotations,
-templates, rows, panels...) as well as a number of datasources and plugins.
-
-We'd like to limit the Grafana plugins supported in Grafonnet to only official plugins.
+Grafonnet aims to support any basic feature of dashboards (annotations,
+templates, rows, panels...) as well as a number of datasources and plugins. That
+is, official plugins only.
```

Comment:
```suggestion
templates, rows, panels...) as well as a number of datasources and plugins. That
is, core Grafana features only.
```

Reply:
Changed to "...core Grafana features and plugins...".